Formula/llvm.rb

```diff
@@ -249,32 +275,17 @@ def install
         cxxflags << "-isystem#{toolchain_path}/usr/include/c++/v1"
         cxxflags << "-isystem#{toolchain_path}/usr/include"
         cxxflags << "-isystem#{macos_sdk}/usr/include"
-
-        extra_args.reject! { |s| s["CMAKE_CXX_FLAGS"] }
-        extra_args << "-DCMAKE_CXX_FLAGS=#{cxxflags.join(" ")}"
-      end
-
-      # On Linux, our just-built Clang needs a little help finding C++ headers,
-      # since we did not build libc++, and we are using the libstdc++ headers instead.
-      # We also need to make sure it links to libstdc++ for brewed GCC rather than
-      # the host GCC which is too old.
-      unless OS.mac?
-        gcc_version = Formula["gcc"].version.major
-
-        cxxflags << "-isystem#{Formula["gcc"].opt_include}/c++/#{gcc_version}"
-        cxxflags << "-isystem#{Formula["gcc"].opt_include}/c++/#{gcc_version}/x86_64-pc-linux-gnu"
+        elseif !OS.mac?
```

Comment:
Not a Rubocop error. It needs to be `elsif` not `elseif`.
```suggestion
      elsif !OS.mac?
```